spark/setup_env

```diff
@@ -1,19 +0,0 @@
-#!/bin/bash
-
-if [[ -z "${SPARK_HOME}" ]]
-then
```

Comment:
Why can you delete this file? Not all Hail instances will be run through the service. Or is there another file somewhere else that also performs these checks?